core/network/build.gradle.kts

```diff
@@ -61,6 +61,8 @@ dependencies {
     implementation("io.ktor:ktor-client-json:2.3.4")
     implementation("io.ktor:ktor-client-websockets:2.3.4")
     implementation("io.ktor:ktor-serialization-kotlinx-json:2.3.4")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.0")
```

Comment:
We have serialization plugin in model module and model module should only contains network models. 

Why did you add the serialization in network module?

Reply:
this is done